script/automerge.ts

```diff
@@ -0,0 +1,133 @@
+require('dotenv-safe').load()
+
+import * as Octokit from '@octokit/rest';
+
+const OWNER = 'electron'
+const REPO = 'i18n';
+const BOTNAME = 'glotbot'
+const SEMANTIC_TITLE = 'feat: New Crowdin translations (auto-merging 🤖)';
+const MERGEABILITY_RETRY_WAIT_TIME = 5_000
+
+enum Mergeability {
+  MERGEABLE,
+  UNMERGEABLE,
+  UNKNOWN
+}
+
+const timer = (time: number): Promise<void> => {
+  return new Promise(resolve => setTimeout(resolve, time))
+}
+
+const github = new Octokit({
+  auth: process.env.GITHUB_TOKEN
```

Comment:
One other thing: I think we should use our custom `GH_TOKEN` here because it's owned by the bot and will trigger additional workflows.